app/src/main/java/ch/epfl/qedit/view/edit/EditQuizActivity.java

```diff
@@ -2,19 +2,49 @@
 
 import android.os.Bundle;
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.lifecycle.ViewModelProvider;
 import ch.epfl.qedit.R;
+import ch.epfl.qedit.model.Question;
+import ch.epfl.qedit.model.Quiz;
+import ch.epfl.qedit.view.quiz.QuestionFragment;
+import ch.epfl.qedit.viewmodel.QuizViewModel;
+import java.util.Arrays;
+import java.util.List;
 
 public class EditQuizActivity extends AppCompatActivity {
+    private QuizViewModel model;
+    private Quiz quiz;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_quiz);
 
+        tempQuiz();
```

Comment:
There is a method in the androidTests under Util that return a standard quiz for tests etc. It could be of use for you in the future when need a quiz for testing. By the way didn't you prepared the code that get the quiz from a bundle? I think that this will be the way how we pass around the quiz, like in the QuizActivity.

Reply:
Oh yes, but since nothing is connected with everything else. I created some dummy one. This should be deleted in the future and replaced.